app/src/test/java/de/westnordost/streetcomplete/quests/opening_hours/AddOpeningHoursTest.kt

```diff
@@ -160,6 +163,60 @@ class AddOpeningHoursTest {
         )))
     }
 
+    @Test fun `isApplicableTo returns false for known places with recently edited opening hours`() {
+        assertFalse(questType.isApplicableTo(
+            node(tags = mapOf("shop" to "sports", "name" to "Atze's Angelladen", "opening_hours" to "Mo-Fr 10:00-20:00"), timestamp = currentTimeMillis())
+        ))
+    }
+
+    @Test fun `isApplicableTo returns true for known places with old opening hours`() {
+        val milisecondsFor400Days : Long =  34560000000 // 1000 * 60 * 60 * 24 * 400
```

Comment:
otherwise it overflows into a different value

there is likely a better way to handle this